tests/md.test.ts

```diff
@@ -87,7 +108,151 @@ First paragraph.
       },
       {
         type: 'Paragraph',
-        content: ['First paragraph.']
+        content: [
+          'A paragraph with ',
+          {
+            type: 'Emphasis',
+            content: ['emphasis']
+          },
+          ', ',
+          {
+            type: 'Strong',
+            content: ['strong']
+          },
+          ', ',
+          {
+            type: 'Delete',
+            content: ['delete']
+          },
+          ' and ',
+          {
+            type: 'Verbatim',
+            value: 'verbatim'
+          },
+          ' text.'
+        ]
+      },
+      {
+        type: 'Paragraph',
+        content: ['A paragraph with ', true, ' and ', false, ' boolean values.']
```

Comment:
I'll admit that the conversion of `[x] == true` and `[ ] == false` caught me a bit off guard. Based on my experiences using other markdown editors I expected this to be remain unchanged, as square brackets.

Reply:
Thanks for catching this and pulling me up on it. I was getting a little too far ahead and thinking about how we will represent user changeable variables (ie. inputs) in documents. The checkbox representation could be a nice repr for booleans at various places in the text (not just at the start of list items). But lets come back to that decision in a future iteration. I have removed it for now.